code/modules/vending/autodrobe.dm

```diff
@@ -75,6 +75,7 @@
 		/obj/item/clothing/mask/gas/sexyclown = 1,
 		/obj/item/clothing/under/rank/civilian/clown/sexy = 1,
 		/obj/item/clothing/mask/gas/sexymime = 1,
+		/obj/item/clothing/suit/toggle/suspenders = 1,
```

Comment:
Modularize this by putting it in the autodrobe's `skyrat_products` list in the modular_skyrat folder wherever that is defined